frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -476,7 +476,6 @@ private void prepareDetailView(RenderRequest request, RenderResponse response) {
                 ComponentService.Iface client = thriftClients.makeComponentClient();
                 Component component = client.getComponentById(id, user);
                 final Set<License> licenses = new HashSet<>(SW360Utils.getLicenses(component.getMainLicenseIds(), user.getDepartment()));
```

Comment:
Now that you've removed `component.setMainLicenses(licenses);`, `licenses` is never used anymore